contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,109 @@
+  All pull requests need to be throroughly reviewed before they can be merged into the master branch.
+  Help us get a headstart by participating in the code review process.
+  .. add a howto/chat to join
+  
+  Read more about our **style guide and other requirements**: :ref:`doc_pr_review_guidelines`
+
+- **Write Plugins (GDScript, C#, & more)**
+  Community addons are not directly included in the core engine download or repository, yet they provide essential quality of life upgrades for your fellow game developers.
+  Upload your plugins to the `Godot Asset Library <https://godotengine.org/asset-library/asset>` to make them available to others.
+
+- **Demo projects (GDScript, C#, and making Assets)**
+  We provide new users with `demo projects <https://github.com/godotengine/godot-demo-projects/>` so they can quickly test new features or get familiar with the engine in the first place.
+  At industry events, we might even exhibit these demo projects to showcase what Godot can do! 
+  Help improve existing projects or supply your own to be added to the pool, and join the `Contributor RocketChat <https://chat.godotengine.org/channel/demo-content>` to talk about it.
+
+- **Documentation**
+  The documentation is one of the most essential parts of any tech project, yet the need to document new features and substential often gets overlooked. 
+  Join the `Contributor RocketChat <https://chat.godotengine.org/channel/documentation>` to improve the Godot Engine with your technical writing skills.
+
+- **Translations (spoken languages other than English)**
+  Are you interested in making the Godot Engine more accessible to non-English speakers? 
+  Community-translations are hosted `here <https://hosted.weblate.org/projects/godot-engine/godot/>`
+
+
+Community support
+-----------------
+
+- **Call for Moderators**
+  With a community of our size, we need people to step up as volunteer moderators in all kinds of places. 
+  These teams are organized by the Godot Foundation, but would not function without the dedication of active community members like you.
+
+  Have a look around your favorite community platform and you might come across open application calls. Currently we are looking for:
+  *List under construction*
+
+- **Answer tech-support questions**
+  With many new people discovering the Godot Engine recently, the need for peer-to-peer tech-support has never been greater. 
+  Be it on the `Forum <https://forum.godotengine.org/>`, our `subreddit <https://www.reddit.com/r/godot/>`, or on `Discord <https://discord.gg/bdcfAYM4W9>`, you can always brighten someone's day by helping them get their personal projects back on track.
+
+- **Create tutorials & more**
+  How did you get started with the Godot Engine? 
+  Chances are you looked for learning materials outside of what the documentation provides.
+  Without content creators covering the game development process, there would not be this big of a community today.
```

Comment:
There needs to be a newline at the end of this file, it's why the build is failing.

Reply:
Oh thanks for explaining that!